webui/react/src/pages/ProjectDetails.tsx

```diff
@@ -429,7 +429,11 @@ const ProjectDetails: React.FC = () => {
     );
 
     const actionRenderer: ExperimentRenderer = (_, record) => {
-      return <ContextMenu record={record} />;
+      return (
+        <ContextMenu record={record} onVisibleChange={undefined}>
+          <></>
```

Comment:
question: why is this necessary? I noticed you adding fragments in a lot of places but this seems the weirdest.

Reply:
Yes, React complained to set the default value, but if i set the default value in prop, i got another error from [here](https://github.com/determined-ai/determined/blob/master/webui/react/src/pages/ProjectDetails.tsx#L1011) 
I wasn't sure about these type, so I made it like this. I agree that its weird. We should fix it later

Reply:
You should fix the root issue in this PR instead of accumulating tech debt. In this case the issue is that `children: React.ReactNode;` should be `children?: React.ReactNode;` in `ContextMenu`.

Reply:
ok i'll fix this before merging this. I actually tried this but it didnt work, but I didnt spend much time to investigate this, so let me check